addons/mail/static/src/scss/composer.scss

```diff
@@ -336,6 +341,7 @@ $o-mail-attachment-margin: 5px;
             margin: 0 0 0 8px;
             border-radius: 2px;
             vertical-align: bottom;
+            height: 1.2rem;
```

Comment:
what's the purpose of this? making the text more visible? you should probably use `font-size`, `padding` or `font-weight`?